types/lodash/fp.d.ts

```diff
@@ -299,8 +299,8 @@ declare namespace _ {
         <TResult>(customizer: lodash.CloneWithCustomizer<T, TResult | undefined>): TResult | T;
     }
     type LodashCloneWith2x1<T, TResult> = (value: T) => TResult | T;
-    type LodashCompact = <T>(array: lodash.List<T> | null | undefined) => Array<Exclude<T, null | undefined | false | "" | 0>>;
```

Comment:
FYI: I assume changes in this file were generated using the script? I think the script hasn't run in a while because it doesn't handle some functions correctly anymore. If this is causing errors then it's fine to revert this file and skip running the fp script. Especially since it looks like `templateSettings` don't exist in `lodash/fp`.